lib/kubernetes-deploy/statsd.rb

```diff
@@ -22,5 +22,30 @@ def self.build
       end
       ::StatsD.backend
     end
+
+    module MeasureMethods
+      def measure_method(method_name, metric = nil)
+        unless method_defined?(method_name) || private_method_defined?(method_name)
+          raise NotImplementedError, "Cannot instrument undefined method #{method_name}"
+        end
+
+        unless const_defined?("InstrumentationProxy")
+          const_set("InstrumentationProxy", Module.new)
```

Comment:
why not just `InstrumentationProxy = Module.new`?

Reply:
`dynamic constant assignment (SyntaxError)`